app/models/game.rb

```diff
@@ -63,7 +63,7 @@ def set_defaults
     self.aasm_state ||= "Game under review"
   end
 
-  def self.search(search)
+  def self.search(title)
     Game.where("title ILIKE ?", "%#{search}%")
```

Comment:
How can this pass the tests if `title` param isn't even used in the function? `search` should be `title`